website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,191 @@
+- [Developer experience improvements](/blog/2023/05/30/0.72-metro-package-exports-symlinks#developer-experience-improvements)
+- [Mitigation for TextInput ANRs on Android](/blog/2023/05/30/0.72-metro-package-exports-symlinks#mitigation-for-textinput-anrs-on-android)
+- [Moving New Architecture updates](/blog/2023/05/30/0.72-metro-package-exports-symlinks#moving-new-architecture-updates)
+
+### Breaking Changes
+
+- [Deprecated Component Removals](/blog/2023/05/30/0.72-metro-package-exports-symlinks#deprecated-component-removals)
+- [Package renames](/blog/2023/05/30/0.72-metro-package-exports-symlinks#package-renames)
+
+<!--truncate-->
+
+## Highlights
+
+### Metro Updates
+
+#### New `metro.config.js` Setup
+
+In React Native 0.72, we’ve changed the config loading setup for Metro in React Native CLI. Please update your project’s `metro.config.js` file to match the [template’s version](https://github.com/facebook/react-native/blob/76a42c292de838a0dd537935db792eaa81410b9b/packages/react-native/template/metro.config.js).
+
+:::info
+Please update your config file to following [format](https://github.com/facebook/react-native/blob/76a42c292de838a0dd537935db792eaa81410b9b/packages/react-native/template/metro.config.js). You can also follow the [upgrade-helper](https://react-native-community.github.io/upgrade-helper/?from=0.71.8&to=0.72.0).
+
+These format changes to `metro.config.js` will become required in 0.73. For 0.72, we will log a warning if not updated.
+:::
+
+This moves control over extending the base React Native Metro config into your project, and we’ve cleaned up the leftover defaults. In addition, this means that standalone Metro CLI commands, such as `metro get-dependencies`, will now work.
+
+#### Symlink support (Beta)
+
+Symlink support continues to be one of the top-requested features in Metro and, in React Native 0.72, we are happy to announce pre-stable support for it.
+
+Symlink support enables React Native to work transparently with monorepo setups and pnpm, removing the need for workarounds. See [Enabling pre-stable features](/blog/2023/05/30/0.72-metro-package-exports-symlinks#enabling-pre-stable-features) to use in your app.
+
+#### Package Exports support (Beta)
+
+[Package Exports](https://nodejs.org/api/packages.html#exports) is the modern alternative to the package.json "main" field and provides new capabilities for npm packages to define their public API and target React Native.
+
+By enabling Package Exports support in your Metro config, your app will be compatible with the wider JavaScript ecosystem, including via the new "react-native" community condition. See [Enabling pre-stable features](/blog/2023/05/30/0.72-metro-package-exports-symlinks#enabling-pre-stable-features) to use in your app.
+
+:::tip
+See the [dedicated blog post](https://reactnative.dev/blog/2023/05/08/package-exports-support) to learn more about this feature and our plans for stable rollout.
+:::
+
+#### Enabling pre-stable features
```

Comment:
We are committing to the terminology proposed here: https://github.com/react-native-community/discussions-and-proposals/discussions/652

We are determining these features are in beta. We've determined that "pre-stable" encapsulates all stages before stable, including beta. Since we're just introducing this nomenclature, these features may not completely comply with our process as they use `unstable_` in the API. This will be an exception. 

Where else are we using conflicting terminology?

Reply:
I'm just trying to provide the perspective of an end user reading this blogpost and finding all these different terms. Maybe add a callout to mention what you are saying here, with the proposed nomenclature?

Also, since it's not in effect yet, should we even start treating it as official? isn't it jumping the gun a bit 🤔

Reply:
I think the broad strokes of beta, and pre-stable are terms we will likely commit to. I agree the usage of `unstable` isn't ideal as I don't think it matches the expectation of what beta means. I think this can be something we accept for now. I would want to avoid re-naming the prop and re-releasing Metro at this point

Reply:
I can also use beta whereever we say "pre-stable" since pre-stable is a generalization of beta